### PR TITLE
Improve connect release

### PR DESCRIPTION
**Pull request by mroz22**
- add bump-version script. nicer and easier to mantain
- fix some docs
- list @trezor/connect-common in dependencies

considering more automation:
- bump version script as github job? similar to crowdin etc?

### CODE REVIEW

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -19,7 +19,6 @@
     -   [@trezor/connect](./packages/connect/index.md)
         -   [debugging](./packages/connect/debugging.md)
         -   [dependencies](./packages/connect/dependencies.md)
-        -   [deployment](./packages/connect/deployment.md)
```
Comment (karliatto) [inline docs/SUMMARY.md:22]: You are removing this because now this information is included in `packages/connect/README.md`, right?

Comment (mroz22) [reply docs/SUMMARY.md:22]: yes. this file does not even exist anymore

Comment (karliatto) [reply docs/SUMMARY.md:22]: I see `deployment.md` file was already remove from `develop`

**Review by karliatto — APPROVED** [review]
Great automation help! :robot:

**Pull request merged**